File: rolemadelen/programmers/120839_가위바위보.cpp
#include <string>
using namespace std;

string solution(string rsp) {
    string ans = "";
    
    for(int i=0; i<rsp.size(); ++i) {
        if(rsp[i] == '2') ans += "0";
        else if(rsp[i] == '0') ans += "5";
        else ans += "2";
    }
    
    return ans;
}

File: gamsojji/programmers/beginner/120839_가위바위보.js
function solution(rsp) {
    var answer = '';
    var units = rsp.toString().split("");
    units.forEach( unit => {
        if(unit == 2) answer += 0;
        else if(unit == 0) answer += 5;
        else if(unit == 5) answer += 2;
    });
    return answer;
}